website/content/en/preview/concepts/nodeclaim.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "NodeClaims"
+linkTitle: "NodeClaims"
+weight: 30
+description: >
+  Understand NodeClaims
+---
+
+Karpenter uses NodeClaims to perform node autoscaling based on the requests of unschedulable pods and the
+requirements of existing [NodePool]({{<ref "./nodepools" >}}) and associated [NodeClasses]({{<ref "./nodeclasses" >}}).
+While NodeClaims require no direct user input, as a Karpenter user you can monitor NodeClaims to keep track of
+the status of your nodes in cases where something goes wrong or a node drifts from its intended state.
+
+Karpenter uses NodeClaims as a merged representation of the cloud provider instance and the node on the cluster.
+Karpenter creates NodeClaims in response to provisioning and disruption needs (pre-spin). Whenever Karpenter
+creates a NodeClaim, it asks the cloud provider to create the instance (launch), register the created node
+with the node claim (registration), and wait for the node and its resources to be ready (initialization).
+
+This page describes how NodeClaims relate to other components of Karpenter and the related cloud provider.
+
+If you want to learn more about the nodes being managed by Karpenter, depending on what you are interested in,
+you can either look directly at the NodeClaim or at the nodes they are associated with:
+
+* Checking NodeClaims: If something goes wrong in the process of creating a node, you can look at the NodeClaim
+to see where the node creation process might have stalled. Using `kubectl get nodeclaims` you can see the NodeClaims
+for the cluster and using `kubectl describe nodeclaim <nodeclaim>` you can see the status of a particular NodeClaim.
+For example, if the node is not available, you might see statuses indicating that the node has expired, is empty, or has drifted.
```

Comment:
Empty/expired/drift shouldn't show up in liveness, but launch, registration, and initialization should.
```suggestion
For example, if the node is not available, you might see statuses indicating that the nodeclaim failed to launch, register, or initialize. 
```

Reply:
Done.